dev/fattest.simplicity/src/componenttest/containers/ImageVerifier.java

```diff
@@ -56,10 +56,11 @@ public final class ImageVerifier {
         }
 
         //TODO remove once WebSphere Liberty is updated
-        _expectedImages.add(DockerImageName.parse("gvenzl/oracle-free:23.3-full-faststart"));
+        _expectedImages.add(DockerImageName.parse("alpine:3.17"));
```

Comment:
Not quite sure what the `TODO` means.  It looks like `alpine:3.17` is separated from the next loop below.

Does `alpine:3.17` need to be available for a little longer until all testing has been switched over to `public.ecr.aws/docker/library/alpine:3.17`?  Then these two lines can be removed?

Reply:
Since we have two repositories for Liberty and I don't want to do a combined build I have to put in some temporary code to allow for these kinds of updates. 

In a previous PR in Open Liberty I switched from using `gvenzl/oracle-free` to `ghci.io/gvenzl/oracle-free`.
Until, I could update all references to the original `gvenzl/oracle-free` image in WebSphere Liberty I had to put in this temporary code.  Which is now being removed.

In this PR I am switching from using `alpine:3.17` to `public.ecr.aws/docker/library/alpine:3.17` but still need to allow for WebSphere Liberty to use `alpine:3.17` until I can get it updated as well. 

Hope that makes sense :D